### PR TITLE
chore: 0.1.2 release

### PR DESCRIPTION
**Pull request by ragingsquirrel3**
Bump up the version to 0.1.2 in pubspec files and add entries to all the CHANGELOG files.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by cshfang — APPROVED** [review]
Lgtm thank you

**Review by haverchuck — APPROVED** [review]

**Comment by haverchuck** [comment]
Please wait to merge until restart fix is in place.

**Review by cshfang — APPROVED** [review]

File: packages/amplify_analytics_plugin_interface/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+## 0.1.2 (2021-04-16)
+
+### Bug Fixes
+
+- fix: handle hot restart (#491)
+
+
```
Comment (Ashish-Nanda) [inline packages/amplify_analytics_plugin_interface/CHANGELOG.md:7]: Nit: extra line

Comment (ragingsquirrel3) [reply packages/amplify_analytics_plugin_interface/CHANGELOG.md:7]: my bad, removing

**Review by Ashish-Nanda — APPROVED** [review]
Looks good 🚀 
Congratulations on your first PR @ragingsquirrel3!

**Pull request merged**